src/main/java/io/sprucehill/zalando/api/nativecart/model/Address.java

```diff
@@ -0,0 +1,107 @@
+package io.sprucehill.zalando.api.nativecart.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * 
+ * @author dipteewarudkar
+ *
+ */
+public abstract class Address extends Base {
+
+	@JsonProperty
+	private String id;	
+	@JsonProperty
+	private String customer_number;
```

Comment:
Please use camel case notation for property names; use the `value` property on the @JsonProperty annotation to specify a different JSON name